Makefile

```diff
@@ -13,18 +13,25 @@
 #   just the fastest or easiest way.
 
 SHELL := /bin/bash
-PREFIX ?=
+PREFIX ?= /
 SYSCONF ?= etc
 INSTALL_DIR = usr/share
 
-PYTHON_SITELIB ?= usr/lib/python2.7/site-packages
-# Note the underscore used instead of a hyphen
-PYTHON_INST_DIR = $(PREFIX)/$(PYTHON_SITELIB)/subscription_manager
-
 OS = $(shell lsb_release -i | awk '{ print $$3 }' | awk -F. '{ print $$1}')
 OS_VERSION = $(shell lsb_release -r | awk '{ print $$2 }' | awk -F. '{ print $$1}')
 OS_DIST ?= $(shell rpm --eval='%dist')
 
+PYTHON ?= python
+ifeq ($(OS),Fedora)
```

Comment:
This seems brittle... How about something like `python -c 'import sys; print("python%s.%s" % sys.version_info[:2])'` ?